doc/files/package.json.md

```diff
@@ -170,6 +170,16 @@ Both email and url are optional either way.
 
 npm also sets a top-level "maintainers" field with your npm user info.
 
+## support
+
+You can specify a URL for up-to-date information about ways to support
+development of your package:
+
+    { "support": "https://example.com/project/support" }
+
+Users can use the `npm support` subcommand to all the dependencies of
```

Comment:
`support` is a potentially confusing term (it made me think of contacting npm's support team).

Maybe `npm sponsor` instead?

Reply:
Also does this command include transitive dependencies? It looks like it does, but I think this is an important thing to call out, as it might encourage wider participation.